zombienet_tests/functional/0001-parachains-pvf.zndsl

```diff
@@ -32,16 +32,6 @@ eve: parachain 2005 is registered within 60 seconds
 one: parachain 2006 is registered within 60 seconds
 two: parachain 2007 is registered within 60 seconds
 
-# Check if network is fully connected.
-alice: reports peers count is at least 15 within 20 seconds
```

Comment:
I would still check that we have connectivity here, even if the logic changed and it's hard to estimate number of peers.

Reply:
https://github.com/paritytech/polkadot/pull/6826#discussion_r1126348601 see this discussion.